pkg/resource/prunercronjob.go

```diff
@@ -0,0 +1,235 @@
+package resource
+
+import (
+	"fmt"
+	"os"
+
+	batchv1 "k8s.io/api/batch/v1"
+	batchapi "k8s.io/api/batch/v1beta1"
+	kcorev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	batchset "k8s.io/client-go/kubernetes/typed/batch/v1beta1"
+	batchlisters "k8s.io/client-go/listers/batch/v1beta1"
+
+	imageregistryapiv1 "github.com/openshift/api/imageregistry/v1"
+	imageregistryv1listers "github.com/openshift/client-go/imageregistry/listers/imageregistry/v1"
+	"github.com/openshift/cluster-image-registry-operator/defaults"
+	"github.com/openshift/cluster-image-registry-operator/pkg/parameters"
+)
+
+var (
+	defaultSuspend                          = false
+	defaultSchedule                         = "*/1 * * * *"
```

Comment:
This is "run every minute" - needs to be changed to something more sensible like `0 0 * * *` (daily at midnight)

Reply:
fixed in current push (i can read cronjob syntax :) )